app/javascript/react/components/PizzaShowContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+
+const PizzaShowContainer = (props) => {
+  const [pizza, setPizza] = useState({ 
+    product_name: "", 
+    cost: null
+  })
+
+  const [brand, setBrand] = useState({})
```

Comment:
don't necessarily need a separate piece of state for the brand, all the info can live in the same piece of state for the pizza. Though you might need a placeholder in your state so the component doesn't error on the initial render
```
pizza ... useState({ brand: {} })
```